protocol/0070-MKTD-market-decimal-places.md

```diff
@@ -70,8 +70,8 @@ Trades of course result in transfers. The amounts transferred (for the trade as
 ## Acceptance criteria
 
 - As a user, I can propose a market with a different precision than its settlement asset
-  - This proposal is valid if the precision is NOT greater than the settlement asset (<a name="0070-MKTD-001" href="#0070-MKTD-001">0070-MKTD-001</a>). For product spot: (<a name="0070-MKTD-016" href="#0070-MKTD-016">0070-MKTD-016</a>)
-  - This proposal is NOT valid if the precision is greater than the settlement asset (<a name="0070-MKTD-002" href="#0070-MKTD-002">0070-MKTD-002</a>). For product spot: (<a name="0070-MKTD-009" href="#0070-MKTD-009">0070-MKTD-009</a>)
+  - This proposal is valid if the precision is NOT greater than the settlement asset precision - position precision (<a name="0070-MKTD-021" href="#0070-MKTD-021">0070-MKTD-021</a>). For product spot: (<a name="0070-MKTD-022" href="#0070-MKTD-022">0070-MKTD-022</a>)
```

Comment:
For spot I don't know what you mean by position precision.

Reply:
according to spot spec 0080, `position_decimal_places` should be used to specify the number of decimal places of the `base_asset` asset when specifying order size.